packages/mdc-textfield/_mixins.scss

```diff
@@ -473,6 +413,8 @@
   // to position the label within the bounds of the inset padding.
   @include mdc-floating-label-float-position($mdc-text-field-textarea-label-position-y, 0%, $mdc-text-field-textarea-label-scale);
   @include mdc-floating-label-shake-animation(textarea);
+  @include mdc-states-base-color(transparent);
```

Comment:
See previous comment RE excluding selectors for states styles rather than overriding

Reply:
Same as before.